pylib/anki/collection.py

```diff
@@ -235,15 +235,22 @@ def autosave(self) -> None:
         elif time.time() - self._last_checkpoint_at > 300:
             self.save()
 
-    def close(self, save: bool = True, downgrade: bool = False) -> None:
+    def close(
+        self,
+        save: bool = True,
+        downgrade: bool = False,
+        backup_folder: str = "",
```

Comment:
Sorry, I know I only just suggested copying the "empty string = undefined" approach from OpenCollection(), but it looks like we can do this in a better way now:

https://github.com/ankitects/anki/commit/6941bccde4212a15816d07852ab4f6e0ef6395c0

We should be able to pass a str | None into _backend.close_collection(), as the message constructor will accept a None argument for an optional field.

Reply:
No problem. It's great to be able to ditch these nasty wrappers. Too bad that the Python implementation is still so cumbersome.

Reply:
This seems to break passing keywords directly into an RPC.
04b209c